src/customTypes/customTypesMillau.ts

```diff
@@ -0,0 +1,104 @@
+// Copyright 2019-2020 @paritytech/bridge-ui authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+/* eslint-disable sort-keys */
+
+export default {
```

Comment:
Can we load this from a JSON file to avoid duplicating the same content between here and the bridges repo?

Reply:
i like this idea ....